RealmSwift/Sync.swift

```diff
@@ -515,6 +515,36 @@ public typealias Provider = RLMIdentityProvider
     case manual(errorHandler: ErrorReportingBlock? = nil)
 }
 
+/**
+ A configuration controlling how the initial subscriptions are populated when a Realm file is first opened.
+
+ - seeAlso ``Realm.subscriptions``
+ - seeAlso https://www.mongodb.com/docs/atlas/device-sdks/sdk/swift/sync/flexible-sync/ for more information
+   about managing subscriptions with flexible sync.
+ */
+@frozen public struct InitialSubscriptionsConfiguration: Sendable {
```

Comment:
There doesn't appear to be any need for this wrapper as it's not exposing a different API from the obj-c type.

Reply:
Right. I originally planned for this to be constructible and settable on the `SyncConfiguration` class, but then abandoned it because it would require unfreezing the sync config.